packages/react-devtools-core/webpack.standalone.js

```diff
@@ -38,14 +38,16 @@ const babelOptions = {
 
 module.exports = {
   mode: __DEV__ ? 'development' : 'production',
-  devtool: __DEV__ ? 'cheap-module-eval-source-map' : 'source-map',
+  devtool: __DEV__ ? 'cheap-source-map' : 'source-map',
```

Comment:
Oh, this is interesting :)

Not opposed to this change at all. Just wondering why/if it was necessary?

Reply:
oops sorry this was unintentional, i was just testing the effects of it since i saw you did it in #22263